sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_import_test_live.cpp

```diff
@@ -30,6 +30,7 @@ TEST_F(KeyVaultKeyClient, ImportKey)
   JsonWebKey key;
   key.KeyType = KeyVaultKeyType::Rsa;
   // Values from https://docs.microsoft.com/rest/api/keyvault/importkey/importkey
+  // cspell:disable
```

Comment:
There apparently are multiple mechanisms for disabling cspell - is cspell::disable the preferred construct (over `spell_checker` for example)?

Reply:
Their doc says they're equivalent, https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker. I've seen `cspell` used in our sources.